detection-rules/attachment_legal_document_threatening_filename.yml

```diff
@@ -0,0 +1,76 @@
+name: "Attachment: Legal Document with Threatening Filename"
+description: |
+  Detects malicious attachments with legal/threatening language in both the subject line 
+  and filename, often sent with minimal or no body text to appear more official.
+  Common pattern for malware delivery via legal-themed social engineering.
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and (
+    // subject contains legal/case language
+    regex.icontains(subject.subject,
+                    '\b(case\s+summary|court\s+action|legal\s+action|court\s+order|warrant|violation|fine|penalty|demand\s+notice|legal\s+notice|court\s+notice|judgment|tribunal|hearing|proceedings|prosecution|contempt|breach|default|liability|damages|settlement|injunction|summon|subpoena|lawsuit|litigation)\b'
+    )
+    or strings.icontains(subject.subject, 'case summary')
+    or strings.icontains(subject.subject, 'court action')
+    or strings.icontains(subject.subject, 'legal action')
+    or strings.icontains(subject.subject, 'court order')
+    or strings.icontains(subject.subject, 'legal notice')
+  )
+  and (
+    // external sender (not from trusted legal domains)
+    sender.email.domain.root_domain not in $org_domains
+    and sender.email.domain.root_domain not in $high_trust_sender_root_domains
+    and sender.email.domain.root_domain not in ("gov", "courts.gov", "uscourts.gov", "judiciary.gov", "co.za")
+  )
+  and length(attachments) >= 1
+  and any(attachments,
+          // Microsoft Office documents commonly used for malware
+          .file_extension in ("doc", "docx", "pdf", "xls", "xlsx")
+          and (
+            // filename contains threatening/legal language
+            regex.icontains(.file_name,
+                            '\b(court|legal|action|imminent|urgent|notice|summon|subpoena|lawsuit|litigation|judgment|warrant|violation|fine|penalty|demand|tribunal|hearing|proceedings|prosecution|contempt|breach|default|liability|damages|settlement|injunction)\b'
```

Comment:
might be able do do something like court|legal and action|summon|subponena|....

i'd focus on combinations of keywords from observed samples.